rules/windows/persistence_registry_uncommon.toml

```diff
@@ -10,7 +10,7 @@ Detects changes to registry persistence keys that are uncommonly used or modifie
 an indication of an adversary's attempt to persist in a stealthy manner.
 """
 from = "now-9m"
-index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+index = ["logs-endpoint.events.*"]
```

Comment:
We can support this, but we would need to add a HKU key for every HKEY_USERS on the query